notascript/src/App.js

```diff
@@ -1,32 +1,63 @@
-import React, { Component } from "react";
-import OutputStream from "./OutputStream";
-import Buttons from "./Buttons";
+import React, { Component } from 'react';
+import Header from './Header';
+import Footer from './Footer';
+import Buttons from './Buttons';
+import styled from 'styled-components';
+
+const StyledOutput = styled.section`
+	border: 1px solid black;
+	font-size: 3em;
+	max-width: 100vw;
+	word-break: break-all;
+`;
 
 export default class App extends Component {
-  state = {
-    cards: [],
-    target: undefined
-  };
+	state = {
+		outputStream: [],
+		actualSign: ''
+	};
+
+	handleButtonClick(label) {
+		console.log('handleButtonClick', label);
+		const content = label;
+
+		this.setState({
+			actualSign: content,
+			outputStream: [...this.state.outputStream, content]
+		});
+		if (this.state.outputStream.length % 4 === 0) {
+			this.sliceArray(this.state.outputStream);
+		}
+	}
 
-  componentDidMount() {
-    fetch("/cards")
-      .then(res => res.json())
-      .then(data => this.setState({ cards: data }))
-      .catch(error => console.log(error));
-  }
+	sliceArray(array) {
+		let slicedArray = [];
 
-  render() {
-    const { cards } = this.state;
-    const { target } = this.state;
+		for (let i = 0; i < array.length; i += 4) {
```

Comment:
Vielleicht das 'for' noch umbauen? Mir war so, dass for mega ungern gesehen wird.